### ISSUE TITLE
failure in rdmacm/test_md.sockaddr_accessibility/0

### ISSUE THREAD

**Issue by brminich** [openucx/ucx#3009]
http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8156/label=vulcan-jenkins,worker=0/console

```
13:54:31 [----------] 3 tests from rdmacm/test_md
13:54:31 [ RUN      ] rdmacm/test_md.sockaddr_accessibility/0
13:54:31 [     INFO ] Testing ib0 with 65.65.65.0:0
13:54:31 /scrap/jenkins/workspace/hpc-ucx-pr-5/label/vulcan-jenkins/worker/0/contrib/../test/gtest/uct/test_md.cc:505: Failure
13:54:31 Value of: uct_md_is_sockaddr_accessible(md(), &sock_addr, UCT_SOCKADDR_ACC_LOCAL)
13:54:31   Actual: false
13:54:31 Expected: true
13:54:31 
[  FAILED  ] rdmacm/test_md.sockaddr_accessibility/0, where GetParam() = "rdmacm" (1 ms)
13:54:31 [ RUN      ] rdmacm/test_md.rkey_ptr/0
13:54:31 [     SKIP ] (allocation+direct access is not supported by rdmacm)
13:54:31 [       OK ] rdmacm/test_md.rkey_ptr/0 (0 ms)
13:54:31 [ RUN      ] rdmacm/test_md.reg_perf/0
13:54:31 [     SKIP ] (registration is not supported by rdmacm)
13:54:31 [       OK ] rdmacm/test_md.reg_perf/0 (1 ms)
13:54:31 [----------] 3 tests from rdmacm/test_md (2 ms total)
```

**Comment by yosefe** [comment]
closing, probably setup issue
